src/ui.h

```diff
@@ -38,7 +38,7 @@ struct uimenu_entry {
     std::string txt;      // what it says on the tin
     nc_color hotkey_color;
     nc_color text_color;
-    mvwzstr extratxt = { 0, c_unset, "", 0};
+    mvwzstr extratxt = mvwzstr { 0, c_unset, "", 0};
```

Comment:
As this is evidently fragile depending on the compiler, how about you just put it in the constructors?